### PR TITLE
users.md rewording

### PR DESCRIPTION
**Pull request by lucasrpatten**
Reworded small sections of users.md to be clearer and more professional. Added alt tags to images.

### CODE REVIEW

**Comment by williewillus** [comment]
your commit seems to revert several changes made in trunk, can you undo those?

File: docs/users.md [hunk 1]
```diff
@@ -11,44 +11,42 @@ As of current knowledge, fractureiser specifically tagets Linux and Windows inst
...
 
-Upon doing so, if applicable, check your `systemctl` for any changes you may not recognize. Afterwards, follow onto the next section of this page that is relevant to you.
-
-## MacOS Information
-
-The malware does not seem to affect MacOS, so you should be fine. *Recheck this doc in the future if this changes*
+After deleting these files, if applicable, check your `systemctl` for any changes you do not recognize. Then, proceed to the relevant section below based on your situation.
 
-### Scripts
+### Automation Script
```
Comment (aaronliu0130) [inline docs/users.md:39]:
```suggestion
### Scripts
```
I'm not knowledgeable but I don't think this level of verbosity is needed, just "Scripts" should be fine and professional enough, in fact this makes it awkward, and slightly incorrect as there are multiple scripts.

File: docs/users.md [hunk 2]
```diff
@@ -88,9 +88,9 @@ Because of its behavior, we are **very confident** this is a **targeted attack a
 
 The affected accounts had two-factor authentication enabled. This is not a simple password compromise situation. Multiple accounts are affected.
 
-*At this point we cannot be confident claiming any hosting service is unaffected*. Please exercise caution regardless of what site you use. Even Maven repositories may be infected, and this malware goes back months.
+~~Currently, we do not suspect other platforms such as Modrinth to be affected.~~ *At this point we cannot be confident claiming any hosting service is unaffected*. Please exercise caution regardless of what site you use. Even Maven repositories may be infected, and this malware goes back months.
```
Comment (aaronliu0130) [inline docs/users.md:91]:
```suggestion
* At this point we cannot be confident claiming any hosting service is unaffected*. Please exercise caution regardless of what site you use. Even Maven repositories may be infected, and this malware goes back months.
```
This was not said before and there still isn't much suspicion on Modrinth, this wasn't a changed stance, there's no reason I can think of to single out Modrinth. Also, this ruins the list formatting.

File: docs/users.md [hunk 3]
```diff
@@ -88,9 +88,9 @@ Because of its behavior, we are **very confident** this is a **targeted attack a
...
 
-Right now, the malware is dormant due to the loss of its C&C (Command and Control) server and the Stage 0 (what was distributed via mods and modpacks) not having a way to get a new server. If you were infected with Stage 2 (the file described below, dropped by Stage 1 when C&C was up), then **the malware is still active.**
+Right now, the malware is dormant due to the loss of its C&C (Command and Control) server and the Stage0 (what was distributed via mods and modpacks) not having a way to get a new server. If you were infected with Stage2 (the file described below, dropped by Stage1 when C&C was up), then **the malware is still active.**
```
Comment (aaronliu0130) [inline docs/users.md:93]: Like I said above, there should be internal consistency on how we word stages. We can't say "Stage 0" and "Stage0" at the same time, plus I don't see any reason to remove the spaces in the first place.

**Review by aaronliu0130 — CHANGES_REQUESTED** [review]
Note that I'm not affiliated with this repository, I just stumbled upon this

**Comment by unascribed** [comment]
Makes wording less consistent and removes intentional indentation and an entire section.

**Pull request closed without merging**